lib/qme/map/map_reduce_executor.rb

```diff
@@ -186,7 +186,7 @@ def calculate_cv_aggregation
         cv_pipeline = build_query
         cv_pipeline.first['$match']["value.#{QME::QualityReport::MSRPOPL}"] = {'$gt'=>0}
         # Don't include patients that are in MSRPOPLEX
-        cv_pipeline.first['$match']["value.#{QME::QualityReport::MSRPOPLEX}"] = {'$lt'=>1}
+        #cv_pipeline.first['$match']["value.#{QME::QualityReport::MSRPOPLEX}"] = {'$lt'=>1}
```

Comment:
We should think about deleting this line (and the comment above), rather than commenting it out.